public/components/toolbar/spatial_filter/draw_filter_shape.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { Fragment, useEffect, useRef } from 'react';
+import { IControl, Map as Maplibre } from 'maplibre-gl';
+import MapboxDraw from '@mapbox/mapbox-gl-draw';
+import { Feature } from 'geojson';
+import { DrawFilterProperties, FILTER_DRAW_MODE} from '../../../../common';
+
+interface DrawFilterShapeProps {
+  filterProperties: DrawFilterProperties;
+  map: Maplibre;
+  updateFilterProperties: (properties: DrawFilterProperties) => void;
+}
+export const DrawFilterShape = ({
+  filterProperties,
+  map,
+  updateFilterProperties,
+}: DrawFilterShapeProps) => {
+  const onDraw = (event: { features: Feature[] }) => {
+    updateFilterProperties({
+      mode: FILTER_DRAW_MODE.NONE,
+    });
+  };
+  const mapboxDrawRef = useRef<MapboxDraw>(
+    new MapboxDraw({
+      displayControlsDefault: false,
+    })
+  );
+
+  useEffect(() => {
+    if (map) {
+      map.addControl((mapboxDrawRef.current as unknown) as IControl);
```

Comment:
Shall we explicitly add the position 'top-right' of filter draw place to the second parameter?

```
addControl(control: IControl, position?: ControlPosition): this;
```

Reply:
Ack